x-pack/plugins/screenshotting/server/browsers/chromium/driver_factory/metrics.ts

```diff
@@ -11,6 +11,7 @@ import { PerformanceMetrics } from '../../../../common/types';
 
 declare module 'puppeteer' {
   interface CDPSession {
+    // @ts-expect-error required to augment the actual definition from puppeteer
```

Comment:
@eokoneyo do you see any alternatives here? Looks like if this is working, it's due to luck.

Reply:
@tsullivan thanks for your suggestion over slack, it's much cleaner and avoids the augmentation approach.